README.md

```diff
@@ -728,11 +728,6 @@ Each host has a [secrets](https://github.com/Mic92/dotfiles/tree/master/nixos/ev
 
 ## Known limitations
 
-### Restarting systemd services
-
-Right now systemd services are not restarted automatically.
-We want to implement this in future.
-
 ### Initrd secrets
```

Comment:
Can you extend the `buildkite-agent-builder` example to just have a `restartUnits` value set?